<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedBatteryDialog</class>
 <widget class="QDialog" name="AdvancedBatteryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbInfo">
     <property name="title">
      <string>BAT0</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="10" column="1">
       <widget class="QLabel" name="manufacturerLabelValue">
        <property name="text">
         <string>SANYO</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="lastFullCapacityLabelValue">
        <property name="text">
         <string>5664 mAh</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="modelLabel">
        <property name="text">
         <string>Model:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLabel" name="modelLabelValue">
        <property name="text">
         <string>0123456789</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="serialLabel">
        <property name="text">
         <string>Serial:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLabel" name="serialLabelValue">
        <property name="text">
         <string>0123456789</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="barCodeLabel">
        <property name="text">
         <string>Bar code:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="chemistryLabel">
        <property name="text">
         <string>Chemistry:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="chemistryLabelValue">
        <property name="text">
         <string>LION</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="designCapacityLabel">
        <property name="text">
         <string>Design capacity:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="designCapacityLabelValue">
        <property name="text">
         <string>5616 mAh</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lastFullCapacityLabel">
        <property name="text">
         <string>Last full capacity:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cycleCountLabel">
        <property name="text">
         <string>Cycle count:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="cycleCountLabelValue">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="manufactureDateLabelValue">
        <property name="text">
         <string>2011-09-26</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="manufacturerLabel">
        <property name="text">
         <string>Manufacturer:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="barCodeLabelValue">
        <property name="text">
         <string>0123456789</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="firstUseDateLabel">
        <property name="text">
         <string>First use date:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="firstUseDateLabelValue">
        <property name="text">
         <string>2011-12-17</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="manufactureDateLabel">
        <property name="text">
         <string>Manufacture date:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="designVoltageLabel">
        <property name="text">
         <string>Design voltage:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="designVoltageLabelValue">
        <property name="text">
         <string>10.8 V</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdvancedBatteryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdvancedBatteryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
